pkg/devfile/generator/utils.go

```diff
@@ -619,11 +623,87 @@ func getAllContainers(devfileObj parser.DevfileObj, options common.DevfileOption
 				return nil, err
 			}
 		}
-		containers = append(containers, *container)
+		// Check if there is an override attribute
+		if comp.Attributes.Exists(ContainerOverridesAttribute) {
+			patched, err := containerOverridesHandler(comp, container)
+			if err != nil {
+				return nil, err
+			}
+			containers = append(containers, *patched)
```

Comment:
can we add a test case with attributes to cover this? I think we can add one to `TestGetContainers`

Reply:
Done: https://github.com/devfile/library/pull/157/commits/c0fab847058daa40af093bd0e7f6879c55e7fa67